gating/check/pre_deploy.sh

```diff
@@ -60,10 +60,10 @@ if [[ ! -e phantomjs-2.1.1-linux-x86_64.tar.bz2 ]]; then
   tar -xjf phantomjs-2.1.1-linux-x86_64.tar.bz2
 fi
 
-# FrankZhang: Limit venv python to python2 
+# FrankZhang: Limit venv python to python2
 # to avoid python3 code making syntax error https://rpc-openstack.atlassian.net/browse/RI-458
 if [[ ! -d ".venv" ]]; then
-  virtualenv .venv --python=python2 --always-copy --no-download
+  virtualenv .venv --python=python2 --always-copy --never-download
```

Comment:
It should be ``--no-download`` for anything more recent than Trusty, so perhaps there should be a distro check similar to the one earlier in this script.

Reply:
Ahh, makes sense.  Thanks.